### PR TITLE
Added Johnny-Five missing optional 'freq' parameter for ProximityOption

### PR DESCRIPTION
**Pull request by lucas-sg**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: [Proximity API - Parameters](http://johnny-five.io/api/proximity/#parameters)

### CODE REVIEW

**Review by peterblazejewicz — COMMENTED** [review]
@lucas-sg please do not change white-space type used in tis DT, stick to spaces

**Comment by lucas-sg** [comment]
> @lucas-sg please do not change white-space type used in tis DT, stick to spaces

@peterblazejewicz I hadn't realized, my bad. It's all in spaces instead of tabs now :)

File: types/johnny-five/index.d.ts [hunk 1]
```diff
@@ -685,6 +685,7 @@ export declare class Ping {
 export declare interface ProximityOption {
     pin: number | string;
     controller: string;
+		freq?: number;
```
Comment (peterblazejewicz) [inline types/johnny-five/index.d.ts:688]:
```suggestion
    freq?: number;
```

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@lucas-sg thx!

**Comment by lucas-sg** [comment]
Ready to merge

**Pull request merged**